### PR TITLE
changed readme again

### PR DESCRIPTION
**Pull request by thesavneetkaur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # HTML-CSS-SASS
 Learning HTML, CSS and SASS
+Hello i am a developer
```
Comment (thesimranjeet) [inline README.md:3]: This looks good, but format it well

**Review by thesimranjeet — CHANGES_REQUESTED** [review]
Please look into it @thesavneetkaur

File: README.md (hunk 1 above)
Comment (thesavneetkaur) [reply README.md:3]: okay i have done the changes.

**Review by thesimranjeet — APPROVED** [review]
Looks good to me

**Pull request merged**